### PR TITLE
Change example of accessing the logger to use DI instead of Umbraco.Core.Composer.Current

### PR DESCRIPTION
**Pull request by TwoMoreThings**
Umbraco.Core.Composing.Current is not recommended as per this discussion (among others)
https://our.umbraco.com/forum/umbraco-8/96270-using-umbracohelper-in-a-custom-class-in-v8#comment-304427
so this showed a bad practice.

I'm not 100% my link to Composing is correct so this will need to be verified

regards
t

### CODE REVIEW

File: Getting-Started/Code/Debugging/Logging/index.md [hunk 1]
```diff
@@ -91,27 +91,36 @@ If you are writing classes that inherit from one of these special Umbraco base c
...
-            Current.Logger.Info<MyContentFinder>("Trying to find content for url {RequestUrl}", frequest.Uri);
+            _logger.Info<MyContentFinder>("Trying to find content for url {RequestUrl}", frequest.Uri);
 
             //Do Content Finder Logic...
         }
     }
 }
 ```
 
+You will, of course, need to register your ContentFinder [using a Composer](/documentation/Implementation/Composing/index.md)
```
Comment (sofietoft) [inline Getting-Started/Code/Debugging/Logging/index.md]: I think this is the correct link! 😁 

```suggestion
You will, of course, need to register your ContentFinder [using a Composer](../../../Implementation/Composing/index.md)
```

**Comment by sofietoft** [comment]
I've had a chat with one of our experts on this, and we like the chance you've made!

However, we think that we should still mention the option to use `Current.Logger` if for some reason DI is not an option.
Perhaps we could say something like:
> While using Dependency Injection is the recommended way, it is possible to `Current.Logger` instead, if DI is not an option.

What do you think? 🤔

**Comment by TwoMoreThings** [comment]
Yes, that sounds good

File: Getting-Started/Code/Debugging/Logging/index.md [hunk 2]
```diff
@@ -91,27 +91,36 @@ If you are writing classes that inherit from one of these special Umbraco base c
 
 Then you can access the logging functionality via a special 'Logger' property included in those base classes and use the friendlier syntax of `Logger.Info<T>` to pass the type, if you add a reference to `Umbraco.Core.Logging` as a Using statement.
 
-Outside of these places, eg a ContentFinder or your own custom code, you can get a reference to the logger from the `Umbraco.Core.Composing` namespace, using the `Current.Logger` instance.
+Outside of these places, eg a ContentFinder or your own custom code, you can get a reference to the logger via Dependency Injection.
```
Comment (sofietoft) [inline Getting-Started/Code/Debugging/Logging/index.md]:
```suggestion
Outside of these places, eg a ContentFinder or your own custom code, you can get a reference to the logger via Dependency Injection. While using Dependency Injection is the recommended way, it is possible to Current.Logger instead, if DI is not an option.

The following example uses DI.
```

Comment (TwoMoreThings) [reply Getting-Started/Code/Debugging/Logging/index.md]: Hi Sophie, sorry I missed the notification of your comments. I'm happy with that. Thanks, t

**Comment by sofietoft** [comment]
Thanks @TwoMoreThings ! 😄 

I've added the suggestions - both the link and sentence about it still being possible to use Current.Logger.
Let's get this merged 😁

**Pull request merged**